snapcraft/pluginhandler.py

```diff
@@ -399,6 +398,7 @@ def strip(self, force=False):
         self.notify_stage('Stripping')
         snap_files, snap_dirs = self.migratable_fileset_for('snap')
         _migrate_files(snap_files, snap_dirs, self.stagedir, self.snapdir)
+        _copy_dependencies(self.snapdir)
```

Comment:
Needs tracking and library paths used need to be exported to `env`
hint for @kyrofa